lib/absinthe/resolution.ex

```diff
@@ -235,8 +236,30 @@ defmodule Absinthe.Resolution do
 
   def call(res, _), do: res
 
+  @doc """
+  Get a list of strings representing the path to this field resolution.
+  The index of list items is dropped and the schema type is included.
+
+  ## Examples
+  Given some query:
+  ```graphql
+  query {users { email }}
+  ```
+
+  If you called this function inside a resolver on the users email field it
+  returns a value like:
+
+  ```elixir
+  resolve fn _, _, resolution ->
+    Absinthe.Resolution.path(resolution) #=> ["email", "users", "RootQueryType"]
```

Comment:
The desired output here is probably to stringify the integer so that the information isn't lost. This is also a breaking change so we'll need to release this under a new minor version.

Reply:
I've added back the integers as strings. I would suggest this isn't a breaking change, b/c no one was using the previous behavior b/c it was broken.